.github/workflows/publish_release.yaml

```diff
@@ -0,0 +1,44 @@
+name: Publish Release
+
+on:
+  push:
+    branches:
+      - main
+      - release
+  workflow_dispatch:
+
+concurrency: release
+
+permissions:
+  contents: write
+  issues: write
+  checks: write
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v3
```

Comment:
We can now use the bot for CI/CD and allow for automated merges, check out this workflow for generating a GITHUB_TOKEN:

https://github.com/algorandfoundation/liquid-auth-js/blob/develop/.github%2Fworkflows%2Fpackage-release.yml